onnxruntime/test/python/transformers/bert_model_generator.py

```diff
@@ -26,13 +26,13 @@ def reverse_if(inputs, reverse=False):
         inputs.reverse()
     return inputs
 
-
 def create_bert_attention(input_hidden_size=16,
-                          pruned_num_heads=2,
-                          pruned_head_size=4,
-                          use_float_mask=False,
-                          switch_add_inputs=False):
-    # unsqueeze in opset version 13 has two inputs (axis is moved from attribute to input).
+                          num_heads=2,
```

Comment:
name: pruned_num_heads?

Reply:
num_heads is constant through the paths and is simple,